docs/tasks/administer_cluster_quotas.md

```diff
@@ -142,7 +142,7 @@ following:
 apiVersion: kueue.x-k8s.io/v1alpha2
 kind: ResourceFlavor
 metadata:
-  name: x86
+  name: x86-flavor
```

Comment:
if we add `-flavor` to everything, users might get the impression that it's necessary?

I think we could leave it out for simplicity.

Reply:
You're right... and in this case it's clear the difference... I'd still keep default-flavor so it matches the rest of the pages.